bloggies_frontend/src/UserCard/index.tsx

```diff
@@ -23,7 +23,7 @@ function UserCard({ user }: IProp) {
             {user.display_name}
           </NavLink>
         </Card.Text>
-        <Card.Subtitle>Joined {moment(user.join_date).fromNow()}</Card.Subtitle>
+        <Card.Subtitle>Joined {moment(user.membership_start_date).fromNow()}</Card.Subtitle>
```

Comment:
I think we'll have to render line 26 based on whether the`membership_start_date` is null or not! And probably reword it from "Joined ..." to "Premium member since ..."

Reply:
oh i see, i can update that!